test/e2e/volume_prune_test.go

```diff
@@ -63,6 +63,37 @@ var _ = Describe("Podman volume prune", func() {
 		podmanTest.Cleanup()
 	})
 
+	It("podman prune volume --filter until", func() {
+		session := podmanTest.Podman([]string{"volume", "create", "--label", "label1=value1", "myvol1"})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(0))
```

Comment:
This pattern is anti-recommended, please see #10997. Could you fix it (and all instances) to use the recommended form (`Expect(session).To(Exit(0))`)? Thank you!

Reply:
Ok, will fix

Reply:
Done